gemfirexd/tools/src/dunit/java/com/pivotal/gemfirexd/query/JSONBasicTestsDUnit.java

```diff
@@ -542,7 +542,8 @@ public void testAlterTable() throws Exception {
     assertTrue(i == 6);
   }
 
-  public void testUnsupportedClauses() throws Exception {
+  // This test should be removed as these restrictions have been removed
+  public void _testUnsupportedClauses() throws Exception {
     startVMs(1, 3, 0, null, null);
```

Comment:
The valid tests in this is JSON as PK or index column. Partitioning is allowed, so those can be removed but other two can be retained.